services/blocks/standard/lmdfinalizer/tree/node.go

```diff
@@ -0,0 +1,71 @@
+package tree
+
+import (
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	"github.com/wealdtech/chaind/services/chaindb"
+)
+
+// VoteWeight contains weighted votes towards a block.
+type VoteWeight uint64
+
+// Node in LMD finalizer Tree
+type Node struct {
+	parent *Node
+
+	root       phase0.Root
+	parentRoot phase0.Root
```

Comment:
Is this the same as parent.root?  If so, is it necessary here?

Reply:
Okay, I understand why you'd want to keep it around by itself.  Makes sense if blocks can arrive out of order.